File: app/src/main/java/com/countrydelight/lintruleproject/java_test/variable_scope_test/RightGlobalVariableScopeTest.java
package com.countrydelight.lintruleproject.java_test.variable_scope_test;

import android.content.Context;
import android.view.View;

public class RightGlobalVariableScopeTest extends View {

    private final String rightGlobalVariableScope = "right_scope";

    public RightGlobalVariableScopeTest(Context context) {
        super(context);
    }

    public void rightGlobalVariableScope1() {
        System.out.println(rightGlobalVariableScope.length());
    }

    private void rightGlobalVariableScope2() {
        System.out.println(rightGlobalVariableScope.length());
    }

    @Override
    protected void onSizeChanged(int w, int h, int oldw, int oldh) {
        super.onSizeChanged(w, h, oldw, oldh);
    }
}

File: app/src/main/java/com/countrydelight/lintruleproject/kotlin_test/variable_scope_test/RightGlobalVariableScopeTest.kt
package com.countrydelight.lintruleproject.kotlin_test.variable_scope_test

class RightGlobalVariableScopeTest {

    private var rightGlobalVariableScope = "right_scope"

    fun rightGlobalVariableScope1() {
        print(rightGlobalVariableScope.length)
    }

    fun rightGlobalVariableScope2() {
        print(rightGlobalVariableScope.length)
    }
}